data/RedHat.yaml

```diff
@@ -0,0 +1,4 @@
+---
+tls_checker::package_provider: "puppet_gem"
+tls_checker::tls_checker_path: "/opt/puppetlabs/puppet/bin/tls-checker"
```

Comment:
Maybe it worth it to unconditionally use `puppet_gem` as the provider since I had to rework some code to make it work with the version of Ruby that ships with Debian stretch back when working on these tools…

Reply:
As per our discussion, using the Puppet version of ruby makes the tool less portable. Creating a gem with it and lock the Ruby version may be cleaner but brings more work.

If doing so, when the gem is used from the module, it may be used with Ruby shipped with Puppet ?

I don't know if I'm clear :)

Reply:
The tool is [tls-checker](https://rubygems.org/gems/tls-checker) which is already a regular gem usable outside the scope of this module.  This module is only about setting-it up and managing a crontab that use it.

As far as I am concerned, I don't really care about where the gem is installed / configured: changing the provider for all environments would not affect operations for existing users.  The advantage I see is that regardless of the operating system, I _know_ that `tls-checker(1)` is available as `/opt/puppetlabs/puppet/bin/tls-checker`.